src/interfaces/assistants_web/src/components/Citations/Citation.tsx

```diff
@@ -46,52 +49,74 @@ export const Citation: React.FC<Props> = ({ generationId, citationKey }) => {
           <div>
             {document.tool_name === TOOL_WEB_SEARCH_ID ? (
               <>
-                <Text styleAs="p-xs" className="uppercase dark:text-marble-800">
+                <Text styleAs="p-xs" className={cn('uppercase', dark(lightText), light(text))}>
```

Comment:
Can we put the `dark(lightText), light(text)` into a variable to be reused?

Reply:
good callout